src/walletConnect/index.tsx

```diff
@@ -290,12 +300,14 @@ export async function pair({ uri }: { uri: string }) {
   PAIRING_TIMEOUT = setTimeout(() => {
     logger.warn(`WC v2: pairing timeout`, { uri });
     client.off('session_proposal', handler);
+    client.off('auth_request', handler);
     showErrorSheet();
     analytics.track('New WalletConnect session time out');
   }, 10_000);
 
   // CAN get fired on subsequent pairs, so need to make sure we clean up
   client.on('session_proposal', handler);
+  client.on('auth_request', handler);
```

Comment:
Since auth is a type of pairing, this is ensuring that we handle clearing the session timeout for auth_request too.